com.unity.render-pipelines.core/Runtime/XR/XRView.cs

```diff
@@ -0,0 +1,20 @@
+namespace UnityEngine.Rendering
+{
+    internal readonly struct XRView
+    {
+        internal readonly Matrix4x4 projMatrix;
+        internal readonly Matrix4x4 viewMatrix;
+        internal readonly Rect viewport;
+        internal readonly Mesh occlusionMesh;
+        internal readonly int textureArraySlice;
+
+        internal XRView(Matrix4x4 projMatrix, Matrix4x4 viewMatrix, Rect viewport, Mesh occlusionMesh, int textureArraySlice)
```

Comment:
should we use ref for occlusionMesh?

Reply:
Mesh is class so the ref it not required here, and we don't modify it